src/mero_conf.erl

```diff
@@ -289,6 +289,15 @@ process_value({servers, {elasticache, ConfigList}}) when is_list(ConfigList) ->
                 lists:map(fun get_elasticache_cluster_configs/1, ConfigList)
         end,
     {servers, lists:flatten(HostsPorts)};
+process_value({servers, {mfa, {Module, Function, Args}}}) ->
+    case catch erlang:apply(Module, Function, Args) of
```

Comment:
Do you mind using a `try…of…catch…end` here instead of `case catch`?

Reply:
Done